Gemfile

```diff
@@ -13,24 +13,17 @@ def location_for(place_or_version, fake_version = nil)
   end
 end
 
-#ruby_version_segments = Gem::Version.new(RUBY_VERSION.dup).segments
```

Comment:
This will break our ability to use `pdk update`. But presumably there's no way to do this in https://github.com/puppetlabs/puppetlabs-pam_tools/blob/main/.sync.yml, since I think puppet-module-* is central to their dependency management.

Reply:
Yep, I'm not saying we should merge this; just offering it as one discussion for side-stepping the license issue. I could probably trim this to just remove puppet-module-posix-dev, also, though that may not gain us much.

Reply:
Legal is bringing this to the PDK team to have it addressed more broadly. For us, we have approval from legal to continue as we are, so I'll just note that in Snyk.  No reason to introduce any potential issues with our code by trying to hack around it.

Reply:
:+1: I'll close this, but I'm going to up another pr to switch the github actions license from connect to cd4pe, which will have a better chance of completing with available resources I think.